modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2691,17 +2691,24 @@ def broadcast_apply_full_axis(
             keep_partitioning=keep_partitioning,
         )
         kw = {"row_lengths": None, "column_widths": None}
-        if dtypes == "copy":
+        if isinstance(dtypes, str) and dtypes == "copy":
```

Comment:
What for?

Reply:
so there wouldn't be confusing comparisons:
```python
>>> sr
col1      int64
col2    float64
dtype: object
>>> sr == "copy"
col1    False
col2    False
dtype: bool
```

Reply:
ok